src/components/Ui/Button.vue

```diff
@@ -54,6 +56,10 @@ export default {
     border-color: var(--link-color) !important;
   }
 
+  &.input--warning {
+    border-color: red !important;
```

Comment:
Same as Icon component, this class / props is not necessary, we can use border-red class from PrimerCSS

Reply:
Same here doesn't work without `!important`